weaver/core/relay/config/Dummy_Relay.toml

```diff
@@ -52,6 +53,6 @@ tls=false
 tlsca_cert_path="credentials/fabric_ca_cert.pem"
 [drivers.Dummy]
 hostname="localhost"
-port="9092"
+port="9090"
```

Comment:
Is this change required? It might cause our CI workflow to fail.

Reply:
hmm .. I can change it if you see there is no problem with that. It seems that we use the port 9090 as the default port in `core/drivers/fabric-driver/.env.template`:

`RELAY_ENDPOINT=localhost:9080
RELAY_TLS=false
RELAY_TLSCA_CERT_PATH=path_to_tls_ca_cert_pem_for_relay
DRIVER_ENDPOINT=localhost:9090
DRIVER_TLS=false` 

and in `.env1.n1` etc. 

I did the same for the `.env.satp.template`. Please let me know if it is safe to change the port to 9092 because I might be missing something here.